chart/values.yaml

```diff
@@ -308,11 +318,11 @@ components:
       http:
         expose: true
         containerPort: 80
-        nodePort: 31080
+        # nodePort: 31080
       https:
         expose: true
         containerPort: 443
-        nodePort: 31443
+        # nodePort: 31443
```

Comment:
Why disable the `nodePort`? This is what makes (or at least used to) the load balancer work on gitpod.io.

Reply:
There is a conflict when the service type "ClusterIP" is used. Therefore `nodePort` has to be excluded from the default `values.yaml`. But this is not directly related to my commit. I have removed it.